bot.js

```diff
@@ -1,28 +1,37 @@
 import { config } from 'dotenv';
-import { fetchNextEventForGroup, formatEventMessage } from './meetupHelpers.js';
+import { formatEventMessage, fetchTodaysEvent } from './meetupHelpers.js';
 import { Client, Intents } from 'discord.js';
+import { checkEventIsNovel } from "./discordHelper.js";
 
 config();
 
 const client = new Client({ intents: [Intents.FLAGS.GUILDS] });
 
 const sendAnnouncement = async (channel) => {
-	const event = await fetchNextEventForGroup(34547654);
+	const event = await fetchTodaysEvent();
   
   if (!event) {
-    console.log('No upcoming events found.');
+    console.log('No events scheduled today.');
```

Comment:
This will litter the logs, and this message doesn't make it out to the discord channel anyway.